Feisty/GameDetails/GameDetailsViewController.swift

```diff
@@ -30,20 +33,40 @@ class GameDetailsViewController: UIViewController {
     
     viewModel.getGameData()
   }
+  
+  private func setGameDetails(with gameName: String,
+                              andAppID appID: String,
+                              andPrice price: String,
+                              andShortDescription shortDescription: String,
+                              andDevelopers developers: String,
+                              andPublishers publishers: String) {
 
-  private func setGameDetails(name: String, appID: String) {
-
-    gameNameLabel.text = name
-    lblAppID.text = appID
+    gameNameLabel.text = gameName
+    lblPrice.text = price
+    lblShortDescription.text = shortDescription
+    lblDevelopers.text = developers
+    lblPublishers.text = publishers
 
   }
-
+  
 }
 
 extension GameDetailsViewController: GameDetailsLoadedType {
   
-  func gameDetailsFound(_ withGameName: String, _ andAppID: String) {
-    setGameDetails(name: withGameName, appID: andAppID)
+  func gameDetailsFound(_ withGameName: String,
```

Comment:
This is a step in the right direction but rather have: `withGameName name: String`
because when you call it below, it's not very correct in terms of readability

Reply:
I can't do that, as it's not my written code. It's code generated as a bridge between Objective-C and Swift. This conversion takes place in a file that is generated during build time that I cannot reliably access, if at all.

Reply:
Ahh okay that would explain the names very clearly then. No problem from my side then

Reply:
You could rename this if you really wanted to. You could use NS_SWIFT_NAME which will give thing a different signature in swift. https://developer.apple.com/documentation/swift/objective-c_and_c_code_customization/renaming_objective-c_apis_for_swift